src/app/core/services/http-events.service.ts

```diff
@@ -4,26 +4,26 @@ import { Observable } from 'rxjs';
 
 @Injectable()
 export class HttpServiceEvents{
-    private fireBase = `https://fe-lab-11-team-lemon-project.firebaseio.com/`
-    constructor(private http: HttpClient){ }    
+	private fireBase = `https://fe-lab-11-team-lemon-project.firebaseio.com/`
+	constructor(private http: HttpClient){ }    
 
-    getEvents(request?:string):Observable<any[]> {
-        if(request) {
+	getEvents(request?:string):Observable<any[]> {
```

Comment:
you can use default parameter and avoid if else block
Something like this.
Also why you use Observable<any[]> ??
When you use `any` the whole idea of typescript is lost
```ts
	getEvents(request: string = ''): Observable<Event[]> {
			return this.http.get<Event[]>(`${this.fireBase}/events.json/${request}`)
		}

Reply:
This is my mistake. I will fix it in future commit.